doc/api_package.rst

```diff
@@ -74,6 +74,12 @@
 
     Files the package provides (list of strings).
 
+  .. attribute:: from_repo
+
+    For installed packages and if the information is available it returns Id of repository with `@`
+    prefix from which the package was installed (string). Otherwise it returns id of repository
```

Comment:
What is the repository returned in the "otherwise" case?
I'm confused that if the information about repository is available, the repository id is returned. And if it's not available, repository id is returned as well..

Reply:
I agree that this needs some clarification.

`For installed packages returns id of repository from which the package was installed prefixed with '@' (if such information is available in the history database). Otherwise returns id of repository the package belongs to (@System for installed packages of unknown origin).`

Reply:
I don't know how to formulate it properly, therefore here is an example. @bocekm Please any help with improvement is welcome.
I installed dnf-1-1 from fedora repository, and I have repository update with dnf-2-2.
It means I have 1 installed package dnf-1-1 (dnf-1-1.from_repo == `@fedora`, dnf-1-1.reponame == `@system`).
I have 1 available package dnf-2-2 (dnf-2-2.from_repo == `update`, dnf-2-2.reponame == `update`)

I installed dnf-1-1 by rpm (dnf has no information from which repository), and I have repository update with dnf-2-2.
It means I have 1 installed package dnf-1-1 (dnf-1-1.from_repo == `@system`, dnf-1-1.reponame == `@system`).
I have 1 available package dnf-2-2 (dnf-2-2.from_repo == `update`, dnf-2-2.reponame == `update`)